src/init/AppRunner.ts

```diff
@@ -94,7 +110,7 @@ export class CliRunner {
    * Resolves a path relative to the current working directory,
    * falling back to a path relative to this module.
    */
-  protected resolveFilePath(cwdPath?: string | null, modulePath = ''): string {
+  public resolveFilePath(cwdPath?: string | null, modulePath = ''): string {
```

Comment:
These don't have to be public anymore right?

Reply:
Not strictly required no. Updated.